Modules/Core/SpatialObjects/include/itkSpatialObject.h

```diff
@@ -494,11 +494,13 @@ class ITK_TEMPLATE_EXPORT SpatialObject:
   bool Evaluate(const PointType & point) const
   { return this->IsInsideInWorldSpace(point); }
 
+#if ! defined ( ITK_FUTURE_LEGACY_REMOVE )
```

Comment:
Should this be `ITK_LEGACY_REMOVE` instead of `ITK_FUTURE_LEGACY_REMOVE`?

Reply:
Thanks.  Fixed now.